.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
Git converting lint endings was causing problem with linting. `LF` is totally fine on modern windows.